src/reference/rpc/event/index.md

```diff
@@ -39,32 +32,16 @@ message RequestSubscribe {
 }
 ```
 
----
+#### Parameters for `RequestSubscribe`
```

Comment:
Do you think this is a good way to show the `default` and `required` properties of the parameters? Most of the time, the parameters would be `empty` or `nil` if user don't pass anything.

Reply:
For default values, we should put what it is exactly (if any) and indicate it is an exact value with fencing. Otherwise, we can just put none. 

For required, we can state "yes" or "no."

What do you think? I updated the docs with sample values, since I'm not sure what they are exactly.

Reply:
sounds good